frontend/src/Troops/TroopLevel.jsx

```diff
@@ -71,7 +70,7 @@ function TroopLevel({ troops, level }) {
       >
         <View style={{ flexDirection: 'row', alignItems: 'center' }}>
           <Image source={troopRound} style={styles.roundAvatar} />
-          <Text style={styles.cardText}>{`Level ${level}: ${troops} Troops`}</Text>
+          <Text style={styles.cardText}>{`Level ${level}: ${count} Troops`}</Text>
```

Comment:
please check the count
- if it's 1, it should say Troop (so `1 Troop`)
- if it's more than 1, it should say Troops (so for example `3 Troops`)